core/main/src/service/user_grants.rs

```diff
@@ -79,14 +80,14 @@ pub struct GrantState {
 impl GrantState {
     pub fn new(manifest: DeviceManifest) -> GrantState {
         let saved_dir = manifest.clone().configuration.saved_dir;
-        let device_grant_path = format!("{}device_grants", saved_dir);
+        let device_grant_path = format!("{}/device_grants", saved_dir);
         let dev_grant_store = if let Ok(v) = FileStore::load(device_grant_path.clone()) {
             v
         } else {
             FileStore::new(device_grant_path, HashSet::new())
         };
 
-        let app_grant_path = format!("{}app_grants", saved_dir);
+        let app_grant_path = format!("{}/app_grants", saved_dir);
         let app_grant_store = if let Ok(v) = FileStore::load(app_grant_path.clone()) {
```

Comment:
Not backward compatible with Ripple 1.0

Reply:
Not related to this ticket, but where are we updating the user_grants to persistent storage via FileStore:sync() method ?

Reply:
Created a new ticket for migrating the changes based on discussion with @Vinodsathyaseelan  https://github.com/rdkcentral/Ripple/issues/243

Reply:
If we use Path joining instead of string formatting, would this be compatible in both 1.0 and 2.0? I think generally when we are writing code to make fs paths we should use Path / PathBuf and not fromat!

Reply:
Agreed `FileStore` uses Path during the loading stage but we can also create the path using the same class